### PR TITLE
update to TF v2.6.0

### PR DESCRIPTION
**Pull request by Wrede**
Related to fedn/issue-245
Compute package updated with corresponding predict function supported in tensorflow 2.6.0.
Class labels prediction are obtained by the argmax on continuous predict probabilities scores.

requirements.txt is updated with strict version 2.6.0 to avoid future version conflicts.

### CODE REVIEW

**Comment by Wrede** [comment]
Close issue [fedn/245](https://github.com/scaleoutsystems/fedn/issues/245) once this PR is accepted

File: mnist-keras/requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
-tensorflow<2.6.0
+tensorflow==2.6.0
```
Comment (ahellander) [inline mnist-keras/requirements.txt:1]: I think if you have tested with 2.5, let's change this to 

tensorflow>=2.5

**Review by ahellander — COMMENTED** [review]
There must be code changes in validate.py, right?

**Comment by Wrede** [comment]
I have tried with tf >=2.3.2, see second commit

**Comment by Wrede** [comment]
In total I have tested with version 2.3.2, 2.4.0, 2.5.0 and 2.6.0

**Review by ahellander — APPROVED** [review]
